### PR TITLE
Add owner to group

### PR DESCRIPTION
**Pull request by RPHaynes**
This is the backend that matches the open pull request on front end along with new additions to service layer and models

### CODE REVIEW

**Review by Justin-Sherfey — APPROVED** [review]

File: src/main/resources/application.properties [hunk 1]
```diff
@@ -1,8 +1,8 @@
-spring.datasource.url=jdbc:postgresql://revrelay-1192.cvtq9j4axrge.us-east-1.rds.amazonaws.com:5432/postgres?currentSchema=public
+spring.datasource.url=jdbc:postgresql://revrelay-1192.cvtq9j4axrge.us-east-1.rds.amazonaws.com:5432/postgres?currentSchema=rph
 spring.datasource.username=postgres
 spring.datasource.password=Password123!
 
-spring.jpa.show-sql=true
-spring.jpa.hibernate.ddl-auto=update
+#spring.jpa.show-sql=true
+spring.jpa.hibernate.ddl-auto=create
```
Comment (AaronLitton) [inline src/main/resources/application.properties:6]: Oh no create again, Also line 1 is your schema?

**Review by AaronLitton — CHANGES_REQUESTED** [review]
Your schema settings got pushed,
Lookout for your Database being hit and reset constantly

**Review by AaronLitton — APPROVED** [review]
You're awesome!

**Review by LutherNa — APPROVED** [review]
I already forgot what comments I made. This is fine. Everything is fine.

**Pull request merged**